day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -252,3 +259,48 @@ ifeval::["{context}" == "downgrade"]
 # journalctl -fu etcd.service
 endif::[]
 ----
+
+== Restoring etcd on a static pod
+
+. Stop the etcd pod by moving the pod manifest YAML file to another directory:
++
+----
+$ mv /etc/origin/node/pods/etcd.yaml .
+----
+
+. Clear all old data:
++
+----
+$ rm -rF /var/lib/etcd
+----
```

Comment:
'rm -rF' looks like doesn't work, does 'rm -rf' works as well? thx
# rm -rF /var/lib/etcd
rm: invalid option -- 'F'
Try 'rm --help' for more information.